Development practices/Emails.md

```diff
@@ -0,0 +1,65 @@
+# Guidelines
+
+## Templates
+In most cases we don't need a custom design for the emails. We use [simple HTML templates](https://github.com/mailgun/transactional-email-templates) and modify them slightly with project's theme colors, and add a logo if needed.
+
+If there's a requirement for having a completely custom template consult with someone from the JS team on which libraries to use.
+
+If somebody from the other team is expected to work on emails, please make sure you first set up the Letter opener, prepare the previews for all mailers and instruct your colleague on how to use the [mailer previews](#mailer-previews).
```

Comment:
If we're talking about MJML here theres a much simpler way
https://mjml.io/try-it-live
or 
```bash
mjml input.mjml -o output.html

```

Reply:
This is great for when you're slicing the template, but after that you still need to put the file to the repo, add some erb code and check how everything renders.